File: primenum.py
def prime():
    n = int(input("Enter a number:"))
    count = 0
    
    for i in range(1, n+1):
        if(n % i == 0):
            count +=1

    if(count <= 1):
        print("It is neither prime nor composite.")
    elif(count == 2):
        print("Prime number.")
    else:
        print("Composite.")

prime()

        

File: primenum.c
#include<stdio.h>
int main()
{
    int n, count = 0, i;
    printf("enter a number:");
    scanf("%d",&n);
    
    for(i=1; i<=n; i++)
    {
        if(n%i==0)
        {
            count += 1;

        }
       
    }
    if(count==2)
    {
        printf("%d is prime", n);
    }
    else
    {
        printf("%d is composite",n);
    }
return 0;

}